lang/en.json

```
@@ -203,6 +203,7 @@
 
 "DND5E.ActorWarningInvalidItem": "{itemType} items cannot be added to a {actorType}.",
 "DND5E.ActorWarningSingleton": "Only a single {itemType} can be added to a {actorType}.",
+"DND5E.ActiveEffectAttributeKeyTooltip": "For a list of common keys, see <a href=https://github.com/foundryvtt/dnd5e/wiki/Active-Effect-Guide>the wiki</a>.",
```

Let's stick this in a nested object. Plus we should be able to handle alternate URLs for enchantments because they have a different set of keys.

```suggestion
"DND5E.ACTIVEEFFECT": {
  "AttributeKeyTooltip": "For a list of common keys, see <a href=\"{url}\">the wiki</a>."
},
```

The ordering here also isn't quite alphabetical, so this should move above the "DND5E.ACTIVITY" line (and leave an extra space surrounding it).